### ISSUE TITLE
Multiple Expression titles in components with nested expression component

### ISSUE THREAD

**Issue by tplevko** [KaotoIO/kaoto-next#822]
### Describe the Bug

This occurs e.g. when configuring the loadBalance node with "sticky" type. There are two expression titles: `Correlation Expression` and `Expression`, both providing FieldHintPopover. This might be confusing for users. 

### Steps to Reproduce the Bug or Issue

1. add LoadBalance 
2. select type - "sticky"
3. see the issue

### Screenshots or Videos

![Screenshot from 2024-02-09 14-12-16](https://github.com/KaotoIO/kaoto-next/assets/4180208/1a04dbaf-57c8-46a2-8915-7f2c3ca7771f)

### Platform

- OS: [e.g. macOS, Windows, Linux]
- Browser: [e.g. Chrome, Safari, Firefox]
- Version: [e.g. 0.30.0]

**Comment by tplevko** [comment]
most probably caused by this - https://github.com/KaotoIO/kaoto-next/blob/main/packages/ui/src/components/Form/expression/ExpressionModalLauncher.tsx#L47-L48

**Comment by igarashitm** [comment]
Possibly from a same root cause with #765

**Comment by tplevko** [comment]
You're right @igarashitm, closing as this is a duplicate of https://github.com/KaotoIO/kaoto-next/issues/765
